api/core/recordNotes.js

```diff
@@ -1,4 +1,4 @@
-import { Group, Shape, loadGeometry, saveGeometry } from '@jsxcad/api-shape';
+import { Group, Shape, load, save } from '@jsxcad/api-shape';
```

Comment:
Is the reason for changing these that the only thing loaded or saved is geometry?